src/app/login-module/login-component/login.component.ts

```diff
@@ -0,0 +1,31 @@
+import { LoginService } from "../../services/login-services/login.service";
+import { Component, OnInit } from "@angular/core";
+import { UserLoginDetails } from "../models/userLoginDetails";
+
+@Component({
+  selector: "app-login",
+  templateUrl: "./login.component.html",
+  styleUrls: ["./login.component.css"]
+})
+export class LoginComponent implements OnInit {
+  constructor(private loginService: LoginService) {}
+  userDetails: UserLoginDetails;
+  isInvalid = false;
+  ngOnInit() {
+    this.userDetails = new UserLoginDetails();
+  }
+  tryLogin() {
+    this.loginService.verifyUserCredentials(this.userDetails).subscribe(
+      response => {
+        //temporary, after completeion we will redirect to home page
+        alert("Successful");
+      },
+      err => {
+        this.isInvalid = true;
+      }
+    );
+  }
+  onFormClick() {
```

Comment:
function name should be self descriptive. 
Don't use general names.

Reply:
Done, Changed to loginClick()